sensor_test.py

```diff
@@ -99,13 +115,30 @@ def test_set_sensor_reading_interval(
     ), "Sensor reading did not change after waiting for specified interval"
 
 
+
+def test_set_invalid_sensor_reading_interval(get_sensor_info, set_sensor_reading_interval):
+    
+    log.info("Get original sensor reading interval")
+    original_interval = get_sensor_info().reading_interval
+    invalid_interval = -1
+    log.info("Request reading interval update to invalid interval")
+    set_sensor_reading_interval(invalid_interval)
+    log.info("Check that sensor responds with an error")
+    sensor_info = get_sensor_info()
+    assert sensor_info.reading_interval == original_interval, "Sensor responded with unexpected reading interval"
+    log.info("Get current sensor reading interval")
```

Comment:
Same deal here, the code below isn't doing anything that you haven't already done in the test.